test/services/validator_check_active_service_test.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ValidatorCheckActiveWorkerTest < ActiveSupport::TestCase
+  setup do
+    create(:epoch_wall_clock, epoch: 123)
+  end
+
+  test 'validator with active stake and nondelinquent should be active' do
+    v = create(:validator, :with_score, account: 'account1')
+    create(:validator_history, account: 'account1')
+    create(:validator_block_history, validator: v, epoch: 122)
+
```

Comment:
Please add here `refute v.is_active' to be sure that service changes this value. Similar in other tests.